affected-paths/core/src/main/kotlin/com/squareup/affected/paths/core/CoreOptions.kt

```diff
@@ -65,7 +65,10 @@ public data class CoreOptions @JvmOverloads constructor(
   val changedFiles: List<String> = emptyList(),
 
   /** Auto-injects the "com.squareup.tooling" plugin to all projects in the build */
-  val autoInjectPlugin: Boolean = true
+  val autoInjectPlugin: Boolean = true,
+
+  /** Include any "includeBuild" builds from the current build */
+  val useIncludeBuild: Boolean = true,
```

Comment:
would we ever want this to be false?

Reply:
I don't think _we_ would want it to be false, but someone may not want to include their composite builds and setting this to `false`.